irc.html

```diff
@@ -10,24 +10,30 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    Please note that in order to chat on #minetest-hub you need to
+    ask an operator for voice and to chat on #minetest-dev you 
+    need to be using a real IRC client.<br>
   </p>
   <ul>
     <li>
       <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel for general Minetest engine and games discussion or
+        support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        Development channel, strictly engine or minetest_game development
```

Comment:
Maybe 'Minetest engine' here for consistency with the above?

Reply:
or "strictly for development of official